### DISCUSSION TITLE
Appetite for some PRs for configuring CSS?

### DISCUSSION THREAD

**Discussion by tomlagier** [react-static/react-static#1463] [General]
Was just getting `react-static` set up today and I found that getting Tailwind integrated was a bit of a chore. The existing plugin https://github.com/dev-warner/react-static-plugin-tailwindcss was not working for me, possibly because the Webpack config format changed? That lead me down a bit of a rabbit hole trying to figure out how to modify the existing `css-loader` and `postcss-loader` config.

A couple of things I'd be happy to PR if it seems like they'd be useful:

1) A version bump for `css-loader`. 2.1.1 is a bit out of date. Nothing too major here, but the API has changed and that leads to some tough-to-troubleshoot behavior when trying to configure it. The new options will throw errors when used with the old version of the loader. This might end up breaking other people who are configuring using that version of the loader, so I get if you're not eager to accept that change.

2) A plugin exposing `css-loader` and `postcss-loader` config a little more transparently. Currently you've got to reconcile the existing `react-static` state and the `webpack` config, which can be a bit tough to figure out where you need to update the loader configs and how to do so. I'd create a plugin (maybe `react-static-plugin-css-options`?) that would accept something like:
```
{
   cssLoaderOptions: { ... },
   postcssPlugins: [ ... ],
   postcssOptions: { ... }
}
```
merge them with the existing loader options, and apply those to the right spot in the `webpack` config.

This would have been useful for my case but I understand if it feels like a leaky abstraction and if you'd rather just have people manipulating the `webpack` config directly.

**Comment by SleeplessByte** [comment]
I think having easy to use plugins is better than instructions to manipulate Webpack directly. I'd happily accept all your proposed changes!
